crates/bevy_input/src/mouse.rs

```diff
@@ -2,46 +2,95 @@ use crate::{ElementState, Input};
 use bevy_ecs::{event::EventReader, system::ResMut};
 use bevy_math::Vec2;
 
-/// A mouse button input event
+/// A mouse button input event.
+///
+/// This event is the translated version of the `WindowEvent::MouseInput` from the `winit` crate.
+/// It is available to the end user and can be used for game logic.
```

Comment:
```suggestion
```

Is it relevant to the end user to say "this comes from the `winit` crate"? I don't feel like so.

Regardless, I think "It is available to the end user and can be used for game logic." doesn't add much value and should be removed.